Makefile

```diff
@@ -219,23 +219,23 @@ define colorecho
 endef
 
 # Get a list of all config targets boards/*/*.px4board
-ALL_CONFIG_TARGETS := $(shell find boards -maxdepth 3 -mindepth 3 -name '*.px4board' -print | sed -e 's|boards\/||' | sed -e 's|\.px4board||' | sed -e 's|\/|_|g' | sort)
+ALL_TARGETS := $(shell find boards -maxdepth 3 -mindepth 3 -name '*.px4board' -print | sed -e 's|boards\/||' | sed -e 's|\.px4board||' | sed -e 's|\/|_|g' | sort)
```

Comment:
`ALL_CONFIG_TARGETS` is IMO more appropriate, as it does not include all Makefile targets.